src/navigation.js

```diff
@@ -30,6 +30,17 @@ window.addEventListener("DOMContentLoaded", () => {
     }
   });
 
+  // Hide Skip to Chat button if Chat is not present
+
+  setTimeout(() => {
+    const skipButton = document.body.querySelector(".skip-to-chat-widget");
+    const chatWidget = document.body.querySelector("iframe[id='launcher']");
+
+    if (!chatWidget) {
+      skipButton.style.display = "none";
+    }
+  }, 500);
```

Comment:
Then, a Help Center can have 2 types of widgets: classic and messenger.

- Messenger seems to support `zE('messenger', 'open')` to open the widget ([docs](https://developer.zendesk.com/api-reference/widget-messaging/web/core/#open)).
- Classic seems to support `zE('webWidget', 'open')` to open the widget ([docs](https://developer.zendesk.com/api-reference/widget/core/#open)).

So we could use a logic like this, to open the widget after clicking the button:

```js
    const skipToWidgetButton = document.querySelector("#skipToWidget");
    if (window.zE) {
     	// web widget is present, add logic to open both the classic and the messenger widget
      skipToWidgetButton.addEventListener("click", () => window.zE(
        window.zE.widget === "classic" ? "webWidget" : "messenger", "open"
      ));
    } else {
      // no web widget is present
      skipToWidget.style.display = "none";
    }
```

What do you think @smansouri? It would also be nice to have @zendesk/emu blessing on this one, to make sure we are not missing something.

Reply:
Thanks Andre, I changed it to use zE which is indeed better than looking for id. However even with this we still need to have some delay since the zE.widget which you see it in this recording.

https://github.com/zendesk/copenhagen_theme/assets/12232798/08f82789-4133-4347-9122-1070255d5599

s after some milliseconds.